gossip/evmstore/carmen.go

```diff
@@ -167,7 +167,8 @@ func (c *CarmenStateDB) SubBalance(addr common.Address, amountInt *big.Int) {
 }
 
 func (c *CarmenStateDB) SetBalance(addr common.Address, amount *big.Int) {
```

Comment:
is to be implemented here:  https://github.com/Fantom-foundation/Sonic/pull/216/files#diff-0f054a649955118fbc88ec9657da6c5fee54def1d6c3d81a56ed0cfde12d4fb8R161